cpp/ps0000/ps0000_sl.cpp

```diff
@@ -1,15 +1,16 @@
-#include <iostream> // importing libraries should be started with # not @ and name is iostream only
+#include <iostream> // importing libraries should be started with # not @ and name is iostream only 2/2
```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`